packages/nodes-base/credentials/Auth0ManagementApi.credentials.ts

```diff
@@ -19,6 +19,7 @@ export class Auth0ManagementApi implements ICredentialType {
 			displayName: 'Session Token',
 			name: 'sessionToken',
 			type: 'hidden',
+			// eslint-disable-next-line n8n-nodes-base/cred-class-field-type-options-password-missing
```

Comment:
Edge case to cover later on, tracked [here](https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/168#issue-1830784996).